ibis/backends/tests/test_join.py

```diff
@@ -175,6 +175,7 @@ def test_mutate_then_join_no_column_overlap(batting, awards_players):
 @pytest.mark.notimpl(["druid"])
 @pytest.mark.notyet(["dask"], reason="dask doesn't support descending order by")
 @pytest.mark.notyet(["flink"], reason="Flink doesn't support semi joins")
+@pytest.mark.skip("risingwave")  # TODO(Kexiang): Risingwave's bug, investigating
```

Comment:
```suggestion
@pytest.mark.broken("risingwave")  # TODO(Kexiang): Risingwave's bug, investigating
```

whatever the reason, we want to get a signal when this test starts to pass, and with `skip`, that change will not be visible.

Reply:
The problem is that this case does not always failed. It actually can pass sometimes. But I haven't got time to explore under what conditions it will fail/pass. I think `skip` is more appropriate for now.